text/0013-Clarity-auction-UI-UX-requirements.md

```diff
@@ -0,0 +1,157 @@
+# Auction UI/UX requirements for Clarity
+
+## Summary
+
+[summary]: #summary
+
+CEP PR: [casperlabs/ceps#0013](https://github.com/CasperLabs/ceps/pull/13)
+
+Clarity is expected to be the main conduit for user interaction with the auction system. This proposal suggests a number of basic monitoring and interactive workflows, with corresponding UI elements, that users should have available at mainnet launch.
+
+## Motivation
+
+[motivation]: #motivation
+
+We anticipate that there would be at least 4 groups of users who may want to interact with the auction system
+
+- Validators, delegators & prospective validators
+- Traders & investors
+- Researchers (including the CasperLabs economics team)
+- Blockchain enthusiasts
+
+It is natural that we should support auction monitoring and interaction workflows for validators, since their nodes form the backbone of the platform. However, we should anticipate the needs of the other groups. 
+
+Traders must have easy means of monitoring the payoffs the platform, which are intimately connected to the auction system, both logically and in the software itself, to encourage correct price formation in the markets. 
+
+Researchers, including the author of the CEP, expect an easily accessible high-level view of platform operation, enabling them to quickly discover unexpected behaviors that need additional consideration in our design and modeling processes. 
+
+Finally, blockchain enthusiasts need to see this critical part of the platform work in real time, using a simple UI, both as an assurance of the correct operation of the platform and as a functional advertisement for the innovation Casper brings to the table.
+
+## Guide-level explanation
+
+[guide-level-explanation]: #guide-level-explanation
+
+We will describe the proposed additions to Clarity from a functional perspective first, providing sample workflows corresponding to each user group in the preceding motivations narrative. The identified protagonist should not be assumed to be the only type of interest in a given scenario (e.g., delegators manage withdrawal purses, too).
+
+We will denote UI elements in *bold* and _italic_ for buttons and display elements, respectively. Certain workflows will require the protagonist to be logged into the CasperLabs signer, assumed to provide key management capabilities similar to Metamask. We will also references "pages," which should be imagined as distinct sets of UI elements, such as Main, Auctions, Analytics and Purses.
+
+### Prospective validator - bidding
+
+A prospective validator needs to *bid* to become one.
+
+    1. Log into the CasperLabs signer.
+    2. Navigate to the Auctions page.
+    3. Submit a *bid,* inducing Clarity to generate a transaction calling into the relevan auction contract entry point with specified parameters.
+    4. Receive confirmation of succesful entry of bid upon finalization, displaying parameters entered and current position in the auction.
```

Comment:
Add: Receive confirmation for 'winning' or 'loosing' the bid?

Reply:
I think we also need to add some indication of the validators personal stake as percentage of their total stake so that those who want to delegate have this data available at their disposal for making delegation decision.